cocos/particle/models/line-model.ts

```diff
@@ -37,7 +37,7 @@ const _vertex_attrs = [
     new Attribute(AttributeName.ATTR_TEX_COORD, Format.RGBA32F), // x:index y:size zw:texcoord
     new Attribute(AttributeName.ATTR_TEX_COORD1, Format.RGB32F), // xyz:velocity
     new Attribute(AttributeName.ATTR_COLOR, Format.RGBA8, true),
-];
+] as (Attribute & { offset: number })[];
```

Comment:
@star-e There is a hack here, which modify Attribute dynamically, `offset` is added to Attribute instance.
Should we add the offset to Attribute class to avoid this hack?

Reply:
The `offset` might not be used and can be removed.